packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -56,7 +56,7 @@ function tokenListIdsToNames(ids: number[]): string {
 
 function TokenLogoFallback() {
   return (
-    <div className="flex h-8 w-8 items-center justify-center rounded-full bg-ocl-blue text-sm font-medium text-white">
+    <div className="flex h-8 w-8 flex-none items-center justify-center rounded-full bg-ocl-blue text-sm font-medium text-white">
```

Comment:
After adding `w-full` to the parent, we have to also add `flex-none` here to prevent side-effects like stretching.